docs(guides/best-practices/dividing-into-systems): first version (#2657)
latticexyz/mud: docs/pages/world/systems.mdx

### Calling one `System` from another

<Callout type="info" emoji="ℹ️">

>   Sometimes you think you need to call a `System` from another, when there is no reason to have separate `System`s at
>   all. [See here](/guides/best-practices/dividing-into-systems) for a discussion of when it makes sense to have a single
>   `System` and when splitting the code makes sense.

Reviewer: ```suggestion
Sometimes there are [easier alternatives than calling a `System` from another](/guides/best-practices/dividing-into-systems).
```
